README.md

```diff
@@ -90,7 +93,10 @@ in the future. Read more about our client support [here](https://github.com/Obol
 Create some testnet private keys for a six node distributed validator cluster with the command:
 
 ```sh
-docker run --rm -v "$(pwd):/opt/charon" ghcr.io/obolnetwork/charon:v0.14.0 create cluster --withdrawal-address="0x000000000000000000000000000000000000dead" --nodes 6 --threshold 5
+# Enter required validator addresses
+WITHDRAWAL_ADDR=<ENTER YOUR WITHDRAWAL ADDRESS HERE>
```

Comment:
can you please add `FEE_RECIPIENT_ADDR` and `--name` to this section too?

Reply:
Added